docfx_yaml/extension.py

```diff
@@ -366,7 +389,20 @@ def find_node_in_toc_tree(toc_yaml, to_add_node):
                 if found_module != None:
                     return found_module
 
-        return None;
+        return None
+
+    def convert_module_to_package_if_needed(obj):
+        if 'source' in obj and 'path' in obj['source'] and obj['source']['path'].endswith(INITPY):
```

Comment:
no need. according to python naming conventions:
https://docs.python.org/2/tutorial/modules.html